.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 Cargo.lock
+
+.idea/
```

Comment:
should be removed from here, just use `.git/info/exclude` locally

Reply:
I think it's a good idea to keep it here, 3 reasons:
1) https://github.com/ClickHouse/ClickHouse/blob/master/.gitignore#L107
2) https://github.com/ClickHouse/clickhouse-rs/pull/252#discussion_r2202988885
3) I used Rustrover (form jetbrains), so if any other uses that (or vscode `.vscode`) files would be generated, can can be committed by mistake.

lmk if you want me to add `.vscode` as well.

Or if you want still want it be removed from `.gitignore`, I'm fine either ways.